packages/terra-demographics-banner/src/_LargeDemographicsBannerDisplay.jsx

```diff
@@ -38,39 +46,69 @@ export default (props) => {
       'demographics-banner',
       'large-demographics-banner',
       { deceased: deceasedDate },
+      { 'is-selectable': isSelectable },
+      { 'is-active': isActive },
       theme.className,
     ),
     customProps.className,
   );
 
   delete customProps.className;
 
+  let profileAvatar;
+
+  if (avatar && avatar.alt && avatar.initials) {
+    profileAvatar = (
+      <Avatar
+        className={cx('profile-avatar')}
+        alt={avatar.alt}
+        color={avatar.color}
+        hashValue={avatar.hashValue}
+        image={avatar.image}
+        initials={avatar.initials}
+        isAriaHidden={avatar.isAriaHidden}
+        isDeceased={!!deceasedDate}
+      />
+    );
+  }
```

Comment:
I understand why this change was made, but it's making things more rigid as we have a lot less control over the lifecycle of the photo state itself. We're prefer to still be able to pass a photo node prop as it lets us refresh the photo more deterministically (for the photo capture workflow) and it also lets us load from a cache easier as well. 

Can you offer us a more flexible way for us to manage photo loading ourselves? I promise we'll render avatar :-)

Reply:
We had an offline meeting and decided to keep the photo prop for now and not replace it with avatar. Consumer shouldn't need to figure out a way to remount just the photo. If needed, we should have a first class way to do that. Since this is more related/involved with consumer's disconnected workflow, we decided that we'll hold off in replacing it with avatar until we do some investigation on #3002 so that we can make the photo more seamless for consumers.